NEWS.md

```diff
@@ -1,3 +1,8 @@
+# moderndive 0.5.3.9001
```

Comment:
Change version number to 0.5.4 here

Reply:
This still needs attention

Reply:
See @mariumtapal 's comment. You want the version number in the NEWS.md file to be 0.5.4.